src/main/java/oss/fosslight/domain/BinaryMaster.java

```diff
@@ -196,13 +196,23 @@ public void setBatWatcher(List<BatWatcher> batWatcher) {
 		this.batWatcher = batWatcher;
 	}
 	public String[] getWatcherDivision() {
-		return watcherDivision;
+		String[] ret = null;
+		if(this.watcherDivision != null) {
+			ret = this.watcherDivision.clone();
+		}
+
+		return ret;

```

Comment:
wouldn't it be better to change it like below?

```java
return watcherDivision.clone();
```

Reply:
Done!

watcherDivision != null ? watcherDivision.clone() : null;

Thank you for your review!

Reply:
Is there any reason to do a null check?

Reply:
I think (null).clone() will cause null point exception.

Is there no case that watcherDivision can be a null?

Reply:
I agree :smile:

Reply:
It is null checked, so I resolve this conversation.